.prettierrc

```diff
@@ -4,5 +4,6 @@
   "printWidth": 70,
   "singleQuote": true,
   "jsxBracketSameLine": true,
-  "arrowParens": "always"
+  "arrowParens": "always",
+  "endOfLine": "auto"
```

Comment:
Can you explain to me more why you need this config

Reply:
this was when prettier was bugging out, but it's all good now, you can remove this redundant config